////////////////////////////////////////////////////////////////////////////////
/// @file Language.proto
///
/// @brief Language protobuf definition.
///
/// Copyright 2017 Bose Corporation
////////////////////////////////////////////////////////////////////////////////

package ProductPb;

message Language
{
    message Properties
    {
        repeated string supported_language_codes  = 1;
    }
    optional string code = 1[ default = "en" ];
    optional Properties properties = 2;
}
